network.go

```diff
@@ -2055,8 +2098,21 @@ func (c *controller) getConfigNetwork(name string) (*network, error) {
 	return n.(*network), nil
 }
 
+func (n *network) lbSandboxName() string {
+	name := "lb-" + n.name
+	if n.ingress {
+		name = n.name + "-sbox"
+	}
+	return name
+}
+
+func (n *network) lbEndpointName() string {
+	return n.name + "-endpoint"
```

Comment:
for the default gw we put a restriction on the max string length, do we need something similar for the lb endpoints or sandbox names?

Reply:
For some reason we restrict the length of the names for gateway *sandboxes* to 12 characters plus the prefix "gateway_".  I'm not sure why we do this for sandboxes only but there doesn't seem to be such a restriction for endpoints, so this should be safe.